package.json

```diff
@@ -8,17 +8,18 @@
   },
   "dependencies": {
     "jaco": "git+https://github.com/muraken720/jaco-js.git",
-    "kuromoji": "0.0.4",
-    "vfile": "^1.1.0"
+    "kuromoji": "0.1.1",
```

Comment:
To use fixed version is better.
Because this library depened on hardcoded `dictDir`.

Reply:
I want to use https://github.com/azu/kuromojin instead of [kuromoji.js](https://github.com/takuyaa/kuromoji.js "kuromoji.js") directly If you are ok.
Because, call multiple [kuromoji.js](https://github.com/takuyaa/kuromoji.js "kuromoji.js")'s `builder` function and load multiple dict. It it a overhead.

kuromojin [cache it](https://github.com/azu/kuromojin/blob/8e21bbce7e4c9daafa13fdec416dfdc2fa72a217/src/kuromojin.js#L26).